actions/appConfig.js

```diff
@@ -55,7 +55,11 @@ class AppConfig {
         this.configMap.certPassword = params.certPassword;
         this.configMap.certKey = params.certKey;
         this.configMap.certThumbprint = params.certThumbprint;
+        this.configMap.enablePreviewPublish = this.getJsonFromStr(params.enablePreviewPublish, []);
         this.configMap.helixAdminApiKeys = this.getJsonFromStr(params.helixAdminApiKeys);
+        this.configMap.bulkPreviewCheckInterval = parseInt(params.bulkPreviewCheckInterval || '30', 10);
+        this.configMap.maxBulkPreviewChecks = parseInt(params.maxBulkPreviewChecks || '30', 10);
+        this.configMap.enablePreviewPublish = this.getJsonFromStr(params.enablePreviewPublish, []);
```

Comment:
Given that there is no publish involved with graybox, I guess this can be just `enablePreview` ? yaml file also should be updated if you are renaming it.

This variable seems to be read twice as well, here and line 58.

Reply:
Done Sunil, removed the Publish references and unused variable from appConfig, added earlier thinking we might need it, removed now

Reply:
Right . There is no requirement to publish at this point. The publish will happen from the source project only. 

That said, a check needs to be added at some point to make sure there is no promote running for a given experience before triggering another promote on the same experience (MWPW-144557)

The `skipInProgressCheck` was added in FG to override that behavior during development in case the promote got stuck and we wanted to reset the status that tracks promote and trigger promote anyway.